src/components/common/Layout/components/Account/components/AccountModal/components/SocialAccount/FarcasterProfile.tsx

```diff
@@ -0,0 +1,318 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  ButtonProps,
+  HStack,
+  Icon,
+  IconButton,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Skeleton,
+  Text,
+  Tooltip,
+  VStack,
+  useBreakpointValue,
+  useDisclosure,
+} from "@chakra-ui/react"
+import useUser from "components/[guild]/hooks/useUser"
+import { usePostHogContext } from "components/_app/PostHogProvider"
+import Button from "components/common/Button"
+import { Modal } from "components/common/Modal"
+import useCountdownSeconds from "hooks/useCountdownSeconds"
+import useSubmit, { SignedValidation, useSubmitWithSign } from "hooks/useSubmit"
+import useToast from "hooks/useToast"
+import { ArrowCounterClockwise, DeviceMobileCamera } from "phosphor-react"
+import rewards from "platforms/rewards"
+import { QRCodeSVG } from "qrcode.react"
+import { useState } from "react"
+import { isMobile } from "react-device-detect"
+import fetcher, { useFetcherWithSign } from "utils/fetcher"
+import DisconnectAccountButton from "./components/DisconnectAccountButton"
+import SocialAccountUI from "./components/SocialAccountUI"
+
+// Polling will only start after this much time
+const APPROVAL_POLL_INITIAL_DELAY_MS = 15_000
+// Delay between polls
+const APPROVAL_POLL_INTERVAL_MS = 5000
+// Poll ends, and a new QR is generated this much before the actual deadline
+const POLL_EARLY_END_SEC = 120
+// Enable the manual QR regenation button, when less than this muc seconds is left
```

Comment:
nit: typo

Reply:
2 typos on one line, nice 😄 
Fixed in https://github.com/guildxyz/guild.xyz/pull/1301/commits/7acc0364a729a924ce682a94bc3ef472f18548a4 but ended up removing this const in https://github.com/guildxyz/guild.xyz/pull/1301/commits/0321fb71d7c04e39b672601fd2599b9c424b1c7c